de4a-connector/src/main/java/eu/de4a/connector/service/spring/Conf.java

```diff
@@ -201,22 +205,30 @@ public RestTemplate restTemplate() {
 	public HttpClient httpClient() {
 		try {
 			LOG.debug("SSL context setted to: {}", sslContextEnabled);
+			SSLConnectionSocketFactory factory;
 			if(Boolean.TRUE.toString().equals(sslContextEnabled)) {
-				SSLConnectionSocketFactory factory = sslConnectionSocketFactory();
-				return HttpClientBuilder.create().setSSLSocketFactory(factory).build();
+				factory = new SSLConnectionSocketFactory(sslContext());
```

Comment:
`sslContextEnabled` should be a boolean 🤣

Reply:
OMG

Reply:
Can you just use `if(sslContextEnabled)`?